friends/src/component/Friends.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Friends = (props) => {
+    return (
+        <div>
+            <h2>Hello.  My name is {props.friend.name}</h2>
+            <h4>My email is {props.friend.email}.</h4>
+            <h4>I am {props.friend.age} years old.</h4>
+        </div>
+    )
+}
+
+Friends.propTypes = {
```

Comment:
Remember for propTypes if you don't mark a prop as `.isRequired` then you want to define some defaultProps